### PR TITLE
Solve the case of the missing ssl_verify

### PR DESCRIPTION
**Pull request by ahills**
Fixes the bad configuration bug I added earlier. Suggestions for adding tests?

### CODE REVIEW

**Comment by ktdreyer** [comment]
@alfredodeza , would you please merge #15 ? 

@ahills you can use the tests in https://github.com/ahills/chacractl/pull/1

**Comment by ahills** [comment]
Tests merged. Thanks @ktdreyer!

File: chacractl/main.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def api_credentials(self):
             user, key = conf_module.user, conf_module.key
         chacractl.config['credentials'] = (user, key)
         chacractl.config['url'] = conf_module.url
-        chacractl.config['ssl_verify'] = conf_module.ssl_verify
+        chacractl.config['ssl_verify'] = conf_module.ssl_verify if hasattr(conf_module, 'ssl_verify') else True
```
Comment (ktdreyer) [inline chacractl/main.py]: You can shorten this to `getattr(conf_module, 'ssl_verify', True)`

Comment (ahills) [inline chacractl/main.py]: Excellent point, thanks. Updated.

**Comment by ktdreyer** [comment]
Would you mind rebasing everything onto the latest master? The rebase operation should remove the merge commits also (they aren't needed here)

**Comment by ktdreyer** [comment]
"Reduce visual complexity" can be squashed/fixedup into the original commit, and a rebase will trigger a TravisCI build as well for the test suite.

**Comment by ahills** [comment]
I hate force pushing, but I can never resist a rebase. Done.

**Comment by ktdreyer** [comment]
Sorry, the tests failed due to the issue described in #20 . Would you please rebase and push again now that that is merged?

**Comment by ktdreyer** [comment]
Ok, great, thanks!

**Comment by ahills** [comment]
Done & passed! :ramen:
